todo-list-app/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+end
+
+def index
+  @todos = Todo.all
+end
```

Comment:
You were headed down the right path, but, needed to put your methods in a separate controller. They should go into the `todos_controller.rb`